src/utils/viewer/viewer.cpp

```diff
@@ -321,21 +322,25 @@ void Viewer::addObject(std::string configFile) {
 
   auto& drawables = sceneGraph_->getDrawables();
 
-  int physObjectID = physicsManager_->addObject(configFile, &drawables);
+  int physObjectID = physicsManager_->addObject(ID, &drawables);
   physicsManager_->setTranslation(physObjectID, new_pos);
 
   physicsManager_->setRotation(physObjectID, esp::core::randomRotation());
 
   objectIDs_.push_back(physObjectID);
 }  // addObject
 
+void Viewer::addObject(std::string configFile) {
```

Comment:
```suggestion
void Viewer::addObject(const std::string& configFile) {
```

since you're changing this function anyway :)

Reply:
Thanks!